components/event-service/internal/events/shared/constants.go

```diff
@@ -34,5 +34,12 @@ const (
 	FieldEventTime        = "event-time"
 	FieldEventType        = "event-type"
 	FieldEventTypeVersion = "event-type-version"
-	FieldData             = "data"
+
+	FieldEventIDV3          = "id"
+	FieldEventTimeV3        = "time"
+	FieldEventTypeV3        = "type"
+	FieldEventTypeVersionV3 = "eventtypeversion"
+	FieldSpecVersionV3      = "specversion"
```

Comment:
The `specversion ` is hard to read, please make it more readable.

Reply:
Actually, it comes from CE [specification](https://github.com/cloudevents/spec/blob/v0.3/http-transport-binding.md#324-examples)